USER_GUIDE.md

```diff
@@ -65,9 +65,24 @@ If you wish to customize your configuration, the plugin can be configured using
 
 ##### Cloud SDK configuration
 
-| Parameter      | Description |
-| -------------- | ----------- |
-| `cloudSdkPath` | Location of the Cloud SDK, the plugin will try to find it if none is specified here. |
+| Parameter          | Description |
+| ------------------ | ----------- |
+| ~~`cloudSdkPath`~~ | Deprecated in favor if `cloudSdkHome` |
+| `cloudSdkHome`     | Location of the Cloud SDK. |
+| `cloudSdkVersion`  | Desired version of the Cloud SDK. (e.g. "192.0.0") |
+
+The cloud sdk will be installed/updated/verified depending on which parameters are configured:
+
+| Parameters Specified   | Action |
+| ---------------------- | ------ |
+| None                   | Latest version of the cloud sdk is downloaded and installed. |
+| Both parameters        | Current cloud sdk installation at `cloudSdkHome` is verified. |
+| `cloudSdkHome` only    | No verification. |
+| `cloudSdkVersion` only | Cloud sdk at specified version is downloaded and installed. |
+
+The cloud sdk is installed in `$USER_HOME/.cache/google-cloud-tools-java/managed-cloud-sdk/<version>/google-cloud-sdk`
+(`%LOCALAPPDATA%/google-cloud-tools-java/managed-cloud-sdk/<version>/google-cloud-sdk` on Windows). 
```

Comment:
I think it's also `/Library/Application Support/...` on macOS?

Reply:
Is this documented somewhere?

Reply:
This was the PR for the OS-specific cache resolution: https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/536

And the issue: https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/534

It looks like it is actually `$USER_HOME/Library/Application Support/...`

Reply:
Thanks